doc/source/installation.rst

```diff
@@ -65,11 +65,22 @@ Base package
 Simulation backends
 ^^^^^^^^^^^^^^^^^^^
 
-We provide multiple simulation backends for Qibo, which are automatically loaded if the corresponding packages are installed, following the hierarchy below:
+We provide multiple simulation backends for Qibo, which are automatically loaded
+if the corresponding packages are installed, following the hierarchy below:
 
-* :ref:`installing-qibotf` (*recommended*): an efficient simulation backend for CPU, GPU and multi-GPU based on TensorFlow custom operators. Install this package if you need to simulate quantum circuits with large number of qubits or complex quantum algorithms which may benefit from computing parallelism.
+* :ref:`installing-qibojit` (*experimental*): an efficient simulation backend for CPU, GPU and multi-GPU based on just-in-time (JIT) compiled custom operators. Install this package if you need to simulate quantum circuits with large number of qubits or complex quantum algorithms which may benefit from computing parallelism.
```

Comment:
qibojit does not support multi-GPU currently so we should probably remove the multi-GPU from this line and put it back once we enable this feature.

Reply:
@stavros11 thanks, I have fixed this issue and resorted the backend priority.